src/ProjectTemplates/Shared/Project.cs

```diff
@@ -272,11 +272,13 @@ public void AssertEmptyMigration(string migration)
         Assert.NotNull(file);
         var contents = File.ReadAllText(file);
 
-        var emptyMigration = @"protected override void Up(MigrationBuilder migrationBuilder)
+        var emptyMigration = @"/// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
         {
 
```

Comment:
Just curious: Why are there blank lines between these braces❔ We don't do that in regular code.